backend/corpora/common/corpora_orm.py

```diff
@@ -1,189 +1,301 @@
+    IN_VALIDATION - Following submission, validate datasets for required metadata and absence of PII.
+    IN_ARTIFACT_CREATION - Following validation, create all Original + Remix matrix formats and cellxgene objects.
+    IN_DEPLOYMENT - The final stage in the pipeline: deploying artifacts to Data Portal and cellxgene applications.
+    """
 
-class LibraryConstructionMethod(Base):
-    __tablename__ = "library_construction_method"
+    NA = "N/A"
+    IN_VALIDATION = "In validation"
+    IN_ARTIFACT_CREATION = "In artifact creation"
+    IN_DEPLOYMENT = "In deployment"
 
-    id = Column(Integer, primary_key=True)
-    name = Column(String(64))
-    created_at = Column(DateTime(True), nullable=False, server_default=DEFAULT_DATETIME)
-    updated_at = Column(DateTime(True), nullable=False, server_default=DEFAULT_DATETIME)
 
+class ValidationState(enum.Enum):
+    """
+    Enumerates DbProject validation states.
 
-class Organ(Base):
-    __tablename__ = "organ"
+    NOT_VALIDATED - Validation not performed yet.
+    VALID - Project is valid.
+    INVALID - Project is invalid.
+    """
 
-    id = Column(Integer, primary_key=True)
-    name = Column(String(32))
-    created_at = Column(DateTime(True), nullable=False, server_default=DEFAULT_DATETIME)
-    updated_at = Column(DateTime(True), nullable=False, server_default=DEFAULT_DATETIME)
+    NOT_VALIDATED = "Not validated"
+    VALID = "Valid"
+    INVALID = "Invalid"
 
 
-class Species(Base):
-    __tablename__ = "species"
+class ProjectLinkType(enum.Enum):
+    """
+    Enumerates DbProject external web link types.
 
-    id = Column(Integer, primary_key=True)
-    name = Column(String(16))
-    created_at = Column(DateTime(True), nullable=False, server_default=DEFAULT_DATETIME)
-    updated_at = Column(DateTime(True), nullable=False, server_default=DEFAULT_DATETIME)
+    PROTOCOL - A link to a sequencing protocol.
+    RAW_DATA - A link to a raw data repository.
+    OTHER - Other.
+    """
 
+    PROTOCOL = "Protocol"
+    RAW_DATA = "Raw data"
+    OTHER = "Other"
 
-class DataRepository(Base):
-    __tablename__ = "data_repository"
 
-    id = Column(Integer, primary_key=True)
-    name = Column(String(32))
-    created_at = Column(DateTime(True), nullable=False, server_default=DEFAULT_DATETIME)
-    updated_at = Column(DateTime(True), nullable=False, server_default=DEFAULT_DATETIME)
+class DatasetArtifactFileType(enum.Enum):
+    """
+    Enumerates DatasetArtifact file types.
 
+    H5AD - An AnnData object describing an expression matrix. Uses the .h5ad extension.
+    RDS - A Seurat file object describing an expression matrix. Uses the .rds extension.
+    LOOM - A AnnData object describing an expression matrix. Uses the .loom extension.
+    CXG - A TileDb object describing a cellxgene object. Uses .cxg extension.
+    """
 
-class Contributor(Base):
-    __tablename__ = "contributor"
+    H5AD = "H5ad"
+    RDS = "Rds"
+    LOOM = "Loom"
+    CXG = "Cxg"
```

Comment:
nit: probably best to keep these all lowercase. The casing weird here.

Reply:
Sure! (I don't _think_ these values get used anywhere but will update)

Reply:
or all caps since they are acronyms. Your choice